source/fuzz/fuzzer_pass_permute_function_variables.cpp

```diff
@@ -61,8 +63,9 @@ void FuzzerPassPermuteFunctionVariables::Apply() {
       }
 
     } while (GetFuzzerContext()->ChoosePercentage(
-        GetFuzzerContext()
-            ->GetChanceOfSwappingAnotherPairOfFunctionVariables()));
+                 GetFuzzerContext()
+                     ->GetChanceOfSwappingAnotherPairOfFunctionVariables()) &&
+             variables.size() > 2);
```

Comment:
This will only return false for a function with 2 variables. @afd, what do you think about limiting the number of iterations of this loop in certain cases? The motivation is that if, say, the function has a small number of variables (2, 3 ...), then we won't apply lots of swaps until we break probabilistically.

Reply:
@Vasniktel yes you are right, it will return false but after applying one transformation, I'm not sure the `RandomIndex` can return the same id twice in the same iteration. If it does I can get reassigned again until `instruction_1_index != instruction_2_index` doing this for function has 2 variables, what do you think?